sdk/keyvault/Azure.Security.KeyVault.Secrets/src/SecretClient.cs

```diff
@@ -160,13 +165,16 @@ public virtual AsyncPageable<SecretProperties> GetPropertiesOfSecretVersionsAsyn
         }
 
         /// <summary>
-        /// Lists the properties of all versions of the specified secret. You can use the returned <see cref="SecretProperties.Name"/> and <see cref="SecretProperties.Version"/> in subsequent calls to <see cref="GetSecret"/>.
+        /// Lists the properties of all enabled and disabled versions of the specified secret. You can use the returned <see cref="SecretProperties.Name"/> and <see cref="SecretProperties.Version"/> in subsequent calls to <see cref="GetSecret"/>.
         /// </summary>
-        /// <remarks>
+        /// <para>
```

Comment:
Did you mean for the `<para>` tags to be contained by `<summary>`?     

If not, please see the comment that I left in `KeyClient` for why I believe this form may be problem.